### PR TITLE
Add support for output of well productivity index

### PR DESCRIPTION
**Pull request by totto82**

### CODE REVIEW

File: opm/output/data/Wells.hpp [hunk 1]
```diff
@@ -58,6 +58,9 @@ namespace Opm {
                 reservoir_water   = (1 << 8),
                 reservoir_oil     = (1 << 9),
                 reservoir_gas     = (1 << 10),
+                productivity_index_water = (1 << 11),
+                productivity_index_oil   = (1 << 12),
+                productivity_index_gas   = (1 << 13),
```
Comment (joakim-hove) [inline opm/output/data/Wells.hpp]: Hehe - we are soon running out of bits ;-)

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/output/data/Wells.hpp (hunk 1 above)
Comment (totto82) [reply opm/output/data/Wells.hpp]: and more is coming...

**Comment by totto82** [comment]
Any reason for this not being merged? I have some more stuff coming up (well potentials) I would prefer to have this merged first.

**Pull request merged**